backend/models/unit_tests/Relevance.test.js

```diff
@@ -0,0 +1,22 @@
+const {
+    getRelevanceScore,
+    strengthQuestion,
+    strengthQuestionResponse,
+    strengthUnrelatedResponse,
+} = require("./unit_testing_constants.cjs");
+
+test("getRelevanceScore(question, matching response)", () => {
+    const relevanceScore = getRelevanceScore(
+        strengthQuestion,
+        strengthQuestionResponse
+    )[1];
+    expect(relevanceScore).toBeGreaterThan(0.7);
+});
+
+test("getRelevanceScore(question, matching response)", () => {
```

Comment:
nit: would be great to differentiate test names by expected result (like question, unrelated response)